packages/web-react/src/components/Tabs/__tests__/TabList.test.tsx

```diff
@@ -1,11 +1,34 @@
 import '@testing-library/jest-dom';
+import { render, screen } from '@testing-library/react';
 import React from 'react';
 import { classNamePrefixProviderTest } from '../../../../tests/providerTests/classNamePrefixProviderTest';
 import { stylePropsTest } from '../../../../tests/providerTests/stylePropsTest';
 import TabList from '../TabList';
+import Tabs from '../Tabs';
 
 describe('TabList', () => {
   stylePropsTest((props) => <TabList data-testid="tab-list-test-id" {...props} />, 'tab-list-test-id');
 
   classNamePrefixProviderTest(TabList, 'Tabs');
+
+  it('should render with custom spacing for each breakpoint', () => {
+    render(
+      <Tabs
+        spacing={{ mobile: 'space-100', tablet: 'space-200', desktop: 'space-300' }}
+        selectedTab="test-tab"
+        toggle={() => {}}
+      >
+        <TabList data-testid="tab-list-test-id">Content</TabList>
```

Comment:
do we need the id here?

Reply:
Unfortunately yes, because stylePropsTest logic works with it